admin-rest-api.yaml

```
@@ -919,6 +1405,29 @@ parameters:
     description: The new partition number or the configurations to be updated for the topic.
     schema:
       $ref: '#/definitions/topic_update_request'
+  
+  entity_name:
+    name: entity_name
+    in: path
+    type: string
+    required: true
+    description: The entity name of the quotas can be `default` or `iam-ServiceId`.
+
+  quotas_create:
+    name: quotas_create
+    in: body
+    required: true       
+    description: The new quotas to be created. 
+    schema:
+      $ref: '#/definitions/quotas_create_request'
+
+  quotas_update:
+    name: quotas_update
+    in: body
+    required: true       
+    description: Existing quotas to be updated. 
+    schema:
+      $ref: '#/definitions/quotas_update_request'
```

Similar to my review comment on `quotas_create` - updates only apply to a single quota.
```suggestion
  quota_update:
    name: quota_update
    in: body
    required: true       
    description: Existing quota to be updated. 
    schema:
      $ref: '#/definitions/quota_update_request'
```